ClientGenerator/src/googleapis/codegen/languages/csharp/default/templates/___package_name___/___package_name___.csproj.tmpl

```diff
@@ -0,0 +1,84 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+
+  <!-- nupkg information -->
+  <PropertyGroup>
+    <Title>{{ api.module.name }} Client Library</Title>
+    <Version>{{ features.currentSupportVersion }}.{{ api.revisionInt16 }}</Version>
+    <PackageVersion>{{ features.currentSupportVersion }}.{{ api.revisionInt16 }}</PackageVersion>
+    <Authors>Google Inc.</Authors>
+    <Copyright>Copyright 2017 Google Inc.</Copyright>
```

Comment:
Likewise here in terms of name - and is updating the year to 2017 the right thing? I can see options of:

- Keep it at 2015
- Update it to 2017
- Make it dynamic